src/components/SearchBar.vue

```diff
@@ -76,6 +77,12 @@ export default {
         timeoutDelay: {
             type: Number,
             default: () => defaults.timeoutDelay
+        },
+
+        /* Set the default search term */
+        defaultTerm: {
+            type:       String,
+            default:    null
```

Comment:
I'm not sure if that might happen here, but we used to have the bug were the `default` value was null and `vue-table` was trying to search for `null` value as a string.
Maybe better to send `''` empty sting?
Just a suggestion. Not sure how relevant it is.